app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    """some explanation"""
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "NotWearingMaskError"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "OutdatedVaccineError"
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return "NotWearingMaskError"

```

Comment:
add a more detailed description

```suggestion
        return "The visitor is not wearing a mask"

```